.github/actions/run-backend-tests/action.yml

```
@@ -132,6 +132,7 @@ runs:
           run: bin/check_kafka_clickhouse_up
 
         - name: Wait for Temporal
+          if: ${{ inputs.segment == 'TEMPORAL' }}
```

"FOSS" and "EE" are only all-uppercase because they're both abbreviations, which "Temporal" is not 😄 
```suggestion
          if: ${{ inputs.segment == 'Temporal' }}
```